docs/config.md

```diff
@@ -703,6 +706,26 @@ intentionally re-using the real message's message ID.
 
 Default: `false`
 
+### `Peering`
```

Comment:
include expected behavior / semantics for how this behaves when one of the two peers has it, and when both have it mutually.